src/CatalystClient.ts

```diff
@@ -24,21 +23,30 @@ import { sanitizeUrl } from './utils/Helper'
 import { ContentClient, DeploymentOptions } from './ContentClient'
 import { LambdasClient } from './LambdasClient'
 import { DeploymentWithMetadataContentAndPointers } from './ContentAPI'
+import { RequestOptions } from 'dcl-catalyst-commons/dist/utils/FetcherConfiguration'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version: VERSION } = require('../package.json')
```

Comment:
Could this be changed to? (and same with the ones below)
```import { version: VERSION } from '../package.json'```

Reply:
Also, what value does this have on tests?

Reply:
I've only added tests for `ContentClient`, and the value of the version is `0.0.0-development`